### PR TITLE
initial purge_haplotigs commit

### PR DESCRIPTION
**Pull request by mroach-awri**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by mroach-awri** [comment]
Request for review @bioconda/core
Thanks!

File: recipes/purge_haplotigs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.1" %}
+{% set build = "0" %}
+{% set tarball = "https://bitbucket.org/mroachawri/purge_haplotigs/get/minimap2.tar.gz" %}
```
Comment (johanneskoester) [inline recipes/purge_haplotigs/meta.yaml]: Looks like the tarball url is not versioned. Can you directly link to a tagged release please?
Also, please only specify the version as a variable. Jinja variables are for parts that change but occur at **multiple** places.

Comment (mroach-awri) [reply recipes/purge_haplotigs/meta.yaml]: Thank you Johannes, I've made the changes. Please let me know if there's anything else I could be doing better as I'm quite new to this.

**Comment by mroach-awri** [comment]
@bioconda/core I've made the requested changes, is this ok to merge?

**Comment by rvalieris** [comment]
LGTM

**Pull request merged**